galsim/interpolatedimage.py

```diff
@@ -435,6 +435,10 @@ def __init__(self, image, x_interpolant=None, k_interpolant=None, normalization=
                 pad_image.image, self.x_interpolant, self.k_interpolant, pad_factor,
                 _force_stepk, _force_maxk, gsparams)
 
+        if (calculate_stepk or calculate_maxk) and sbii.getFlux() == 0.:
```

Comment:
I didn't want to completely disallow using InterpolatedImage with a zero image in case people are using it in a non-standard way where maybe it would be ok.  But the other thing that will mess up if getFlux() == 0 is if the user supplies a given flux value.  So I also raise the exception in that case now.  (Just added: 2c5330e)